src/OpenTelemetry/Metrics/View/MetricView.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="MetricView.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace OpenTelemetry.Metrics
+{
+    public struct MetricView
+    {
+        internal readonly string Name;
+        internal readonly Func<Instrument, bool> Selector;
+        internal readonly IViewRule[] ViewRules;
+        internal readonly Func<IAggregator[]> Aggregators;
+
+        public MetricView(string name, Func<Instrument, bool> selector, Func<IAggregator[]> aggregators, IViewRule[] rules)
```

Comment:
is it possible for the view to be simply matching the proposal in the spec: https://github.com/open-telemetry/opentelemetry-specification/pull/1730/files#diff-32d5f289beed96900fa1e6e69a3ee9cc0b6503ce8883672011c44bc772c0fa2aR92-R105

MetricView(string name, string version,...attribute-keys[], extradimension[]). Accepting Func etc means the configuration cannot be easily done with pure config file.

Reply:
We can add (and only expose) a simplified version that takes in "hard coded" values from the SDK Builder Extension and use existing Func<> versions internally.

Reply:
Also, there is currently no "Spec" that specifies any "pure config file" requirements. If I remember correctly, in last conversation on this topic, most people prefers an API based config. At least as a start.